src/GroceryList.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { db } from './lib/firebase';
+
+function GroceryList(props) {
+  return <div></div>;
+}
+
+export default GroceryList;
```

Comment:
For this component where you want to display a list of entries from FireStore, you can use the `react-firebase-hooks` component to read your data. There is documentation on how to use the hooks in their [Readme file](https://github.com/CSFrequency/react-firebase-hooks/tree/master/firestore#usecollection), specifically you'll want to use the `useCollection` or `useCollectionData` hook. If you have any trouble with getting this configured, feel free to continue updating the branch and I can leave some additional tips!

Reply:
^ I had good luck just using the very example they gave in the documentation for `useCollection`.